src/sanitize/mockData.ts

```diff
@@ -30,7 +30,7 @@ export const PRE_SANITIZED_BALANCE_LOCK = kusamaRegistry.createType(
 	'Vec<BalanceLock>',
 	[
 		{
-			id: 'LockId',
+			id: '00000000',
```

Comment:
What happened here?

Reply:
It expected the id to be of 8 bytes but instead `lockId` is 6 bytes